tools/cli/pkg/printer/table.go

```diff
@@ -276,6 +277,65 @@ func (t *tablePrinter) printEvent(sep string, eventTabWriter io.Writer, e event)
 	return nil
 }
 
-func printOperation(w io.Writer, op string) {
-	fmt.Fprintf(w, " ˫operation %v\n", op)
+func printOperation(w io.Writer, op string, rt runtime.RuntimeDTO) {
+	if rt.Status.Provisioning != nil {
+		if op == rt.Status.Provisioning.OperationID {
+			opStatus := rt.Status.Provisioning.State
+			fmt.Fprintf(w, " ˫%v operation %v: %v\n", "provision", op, opStatus)
+			return
+		}
+	}
+	if rt.Status.Deprovisioning != nil {
+		if op == rt.Status.Deprovisioning.OperationID {
+			opStatus := rt.Status.Deprovisioning.State
+			fmt.Fprintf(w, " ˫%v operation %v: %v\n", "deprovision", op, opStatus)
+			return
+		}
+	}
+	if rt.Status.Update != nil {
+		for _, update := range rt.Status.Update.Data {
+			if op == update.OperationID {
+				opStatus := update.State
+				fmt.Fprintf(w, " ˫%v operation %v: %v\n", "update", op, opStatus)
+				return
+			}
+		}
+	}
+	if rt.Status.UpgradingKyma != nil {
+		for _, upgradeKyma := range rt.Status.UpgradingKyma.Data {
+			if op == upgradeKyma.OperationID {
+				opStatus := upgradeKyma.State
+				fmt.Fprintf(w, " ˫%v operation %v: %v\n", "kyma upgrade", op, opStatus)
+				return
+			}
+		}
+	}
+	if rt.Status.UpgradingCluster != nil {
+		for _, upgradeCluster := range rt.Status.UpgradingCluster.Data {
+			if op == upgradeCluster.OperationID {
+				opStatus := upgradeCluster.State
+				fmt.Fprintf(w, " ˫%v operation %v: %v\n", "cluster upgrade", op, opStatus)
+				return
+			}
+		}
+	}
+	if rt.Status.Suspension != nil {
+		for _, suspension := range rt.Status.Suspension.Data {
+			if op == suspension.OperationID {
+				opStatus := suspension.State
+				fmt.Fprintf(w, " ˫%v operation %v: %v\n", "suspension", op, opStatus)
+				return
+			}
+		}
+	}
+	if rt.Status.Unsuspension != nil {
+		for _, unsuspension := range rt.Status.Unsuspension.Data {
+			if op == unsuspension.OperationID {
+				opStatus := unsuspension.State
+				fmt.Fprintf(w, " ˫%v operation %v: %v\n", "unsuspension", op, opStatus)
+				return
+			}
+		}
+	}
+	fmt.Fprintf(w, " ˫%v operation %v: %v\n", "unknown", op, "use --ops to get details")
```

Comment:
do you think it could make sense to implicitly enable `--ops` by when `--events` is passed?

We could simplify just to catch weird data, which would imho provide slightly more pleasant UX.
```suggestion
	fmt.Fprintf(w, " ˫%v operation %v\n", "unknown", op)
```
if you agree, we should also adjust hints for CLI args here
https://github.com/kyma-project/control-plane/blob/553c9a4355f2ea303b6605c0f04fd472c19753ec/tools/cli/pkg/command/runtime.go#L103

something along the lines of
```
Enables --ops by default when using --events
```

Reply:
Sounds good. I will do so.